### PR TITLE
workflows: add scheduled contributors sync

### PR DESCRIPTION
**Pull request by bobheadxi**
Closes https://github.com/sourcegraph/sourcegraph/issues/26992

See https://github.com/sourcegraph/sourcegraph/pull/37395 for documented new workflow

### CODE REVIEW

**Comment by bobheadxi** [comment]
Current dependencies on/for this PR:
* main
  * **PR #86** <a href="https://app.graphite.dev/github/pr/sourcegraph/clabot-config/86" target="_blank">Graphite</a> 
    * **PR #87** <a href="https://app.graphite.dev/github/pr/sourcegraph/clabot-config/87" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/sourcegraph/clabot-config/87?utm_source=stack-comment).

**Comment by bobheadxi** [comment]
TODO - it seems permissions get reset, might need to do something in the infra repo to provision access

File: .clabot [hunk 1]
```diff
@@ -1,9 +1,8 @@
 {
   "message": "We require contributors to sign our Contributor License Agreement (CLA), and we don't have yours on file. In order for us to review and merge your code, please sign [CLA](https://forms.gle/YnmetmopXNxFxsDUA) to get yourself added.\n\nSourcegraph teammates should refer to [Accepting contirbutions](https://docs.sourcegraph.com/dev/contributing/accepting_contribution) for guidance.",
   "label": "cla-signed",
-  "recheckComment": "The GitHub CLA Bot is rechecking to see that you have [signed the CLA](https://github.com/sourcegraph/clabot-config)...",
+  "recheckComment": "The GitHub CLA Bot is rechecking to see that you have [signed the CLA](https://github.com/sourcegraph/clabot-config) - note that it may take up to 30 minutes for your response to be synchronized.",
```
Comment (jhchabran) [inline .clabot:4]: 🙏

**Review by jhchabran — APPROVED** [review]
@bobheadxi approving so you're not stuck once you've figured out what's going with the permissions. I'm curious about what's causing this too.

**Comment by bobheadxi** [comment]
Nevermind, I was silly and forgot to set `GOOGLE_APPLICATION_CREDENTIALS` 😆 The permissions are fine

**Pull request merged**